tavern/tomes/file_list/main.eldritch

```diff
@@ -1,17 +1,111 @@
+usernfo = sys.get_user()
```

Comment:
Is file_list supposed to be updated in this PR?

Reply:
Yes? I added glob support and permissions